.github/pull_request_template.md

```diff
@@ -3,6 +3,10 @@ Please include a summary of the change and which issue is fixed. Include the mot
 
 Fixes: (link Asana Task)
 
+(Optional: Add a changelog by writing a comment after "#changelog" on the next line)
+
+\#changelog
```

Comment:
@matt-thomason What about the type hint in the changelog
`#changelog [feature|bugfix|refactor] -----`

Reply:
Right now it's not functional, and I didn't have a way of typehinting it without actually writing a comment.
We'll update the functionality of the command and then update the docs. I just want people to start using it :)